boa/verifiers.py

```diff
@@ -203,12 +208,15 @@ def verify(
     if (bundle := get_verification_bundle(contract)) is None:
         raise ValueError(f"Not a contract! {contract}")
 
+    # Set chain_id if verifier supports it
+    if hasattr(verifier, "chain_id"):
+        verifier.chain_id = Env.get_singleton().get_chain_id()
```

Comment:
why does this need to be called, isn't it already set when the verifier is initialized?